src/components/OverviewMap/DetailPanel/DetailPanel.tsx

```diff
@@ -61,51 +63,45 @@ const PanelHeader = styled.div`
   justify-content: space-between;
 `;
 
-const fields = ['created_at', 'state_display', 'category', 'subcategory'];
+interface DetailPanelProps {
+  incident: IncidentSummary;
+  onClose: () => void;
+}
 
-const DetailPanel = ({ incident, onClose }) => (
+const DetailPanel: FC<DetailPanelProps> = ({ incident, onClose }) =>
```

Comment:
Please use `FunctionComponent` instead of `FC`, as this is the new style convention.

Reply:
What convention?